javascript/javascript1/week3/travel.js

```diff
@@ -0,0 +1,21 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function calculateTravelTime(travelInfo) {
+  const { speed, destinationDistance } = travelInfo;
+
+  if (speed <= 0) {
+    return "Error: Speed must be greater than 0";
+  }
+
+  const totalTimeInHours = destinationDistance / speed; // زمان به ساعت

```

Comment:
I would suggest keeping your code, including comments, in English as it is the most accessible. 😄